ssl/d1_srtp.c

```diff
@@ -281,10 +279,9 @@ int ssl_parse_serverhello_use_srtp_ext(SSL *s, PACKET *pkt, int *al)
     SRTP_PROTECTION_PROFILE *prof;
 
     if (!PACKET_get_net_2(pkt, &ct)
-            || ct != 2
-            || !PACKET_get_net_2(pkt, &id)
-            || !PACKET_get_1(pkt, &mki)
-            || PACKET_remaining(pkt) != 0) {
+        || ct != 2 || !PACKET_get_net_2(pkt, &id)
```

Comment:
Not really imho. As I said, this is not a line-by-line beauty contest, and you shouldn't care if the tool gets some things (subjectively) a little worse, so long as impact is net positive.